packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -273,35 +273,35 @@ class SliderThemeData extends Diagnosticable {
     );
   }
 
-  /// The color of the [Slider] rail between the [Slider.min] position and the
+  /// The color of the [Slider] track between the [Slider.min] position and the
   /// current thumb position.
   final Color activeRailColor;
```

Comment:
Should the variables also be renamed?

Reply:
Yes.  Sorry, pilot error. I grepped the first time with -i, but had to redo it, and forgot to add -i...